diff --git a/node_modules/@types/marked/index.d.ts b/node_modules/@types/marked/index.d.ts
index 0cbdf01..db97f38 100644
--- a/node_modules/@types/marked/index.d.ts
+++ b/node_modules/@types/marked/index.d.ts
@@ -90,6 +90,7 @@ declare namespace marked {
 
     class Renderer {
         constructor(options?: MarkedOptions);
+        options: MarkedOptions;
         code(code: string, language: string, isEscaped: boolean): string;
         blockquote(quote: string): string;
         html(html: string): string;
